app/migrations/Version20221128145933.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Mautic\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\Exception\SkipMigration;
+use Mautic\CoreBundle\Doctrine\AbstractMauticMigration;
+use Mautic\PluginBundle\Helper\IntegrationHelper;
+use Mautic\SmsBundle\Form\Type\ConfigType;
+
+final class Version20221128145933 extends AbstractMauticMigration
+{
+    /**
+     * @throws SkipMigration
+     */
+    public function preUp(Schema $schema): void
+    {
+        /** @var IntegrationHelper $integrationHelper */
+        $integrationHelper = $this->container->get('mautic.helper.integration');
+        $integration       = $integrationHelper->getIntegrationObject('Twilio');
+        $settings          = $integration->getIntegrationSettings()->getFeatureSettings();
+        if (empty($settings['disable_trackable_urls'])) {
+            throw new SkipMigration('Schema includes this migration');
+        }
+    }
+
+    public function up(Schema $schema): void
+    {
+        $confFile = dirname(__DIR__).'/config/local.php';
```

Comment:
This path cannot be hard-coded as this file can be named differently or be placed elsewhere. Please use `ParameterLader::getLocalFile()`

Reply:
It is neccesary, because still we need that configFile path to save new param  value

https://github.com/mautic/mautic/blob/02d7f4c4eb454f2367df659fa168598a772df40e/app/migrations/Version20221128145933.php#L41

Reply:
I know. I fear that some instances may not have the config file on that path as it is configurable. So for such instances this migration won't work.

Reply:
Yes, either for us. But what you suggest know?  Use ParameterLader::getLocalFile() and let it like now?

Reply:
Yes, that should give you the same path and if configured differently, it should give you the right file. I can see I mistyped the method name. It should be
```suggestion
        $confFile = Mautic\CoreBundle\Loader\ParameterLoader::getLocalConfigFile(__DIR__.'/../');
```